mappings/17w06a#17w13a.tinydiff

```diff
@@ -2278,12 +2278,12 @@ c	net/minecraft/realms/RealmsButton
 		p	3			p_3
 c	net/minecraft/server/MinecraftServer
 	m	()Lnet/minecraft/C_0890491;	m_8224901		getAdvancements
-c	net/minecraft/C_0008241		net/minecraft/C_0008241
+c	net/minecraft/C_0008241		net/minecraft/entity/living/mob/passive/animal/tamable/ParrotEntity
 	f	Lnet/minecraft/C_6101761;	f_6272576		f_6272576
-	f	Lcom/google/common/base/Predicate;	f_8366956		f_8366956
-	f	Lnet/minecraft/C_0215467;	f_3318642		f_3318642
-	f	Ljava/util/Set;	f_3707826		f_3707826
-	f	Lit/unimi/dsi/fastutil/ints/Int2ObjectMap;	f_1309618		f_1309618
+	f	Lcom/google/common/base/Predicate;	f_8366956		CAN_IMITATE
+	f	Lnet/minecraft/C_0215467;	f_3318642		TAMING_ITEM
+	f	Ljava/util/Set;	f_3707826		EDIBLE_ITEMS
+	f	Lit/unimi/dsi/fastutil/ints/Int2ObjectMap;	f_1309618		SOUND_EVENT_MAP
```

Comment:
The `_MAP` suffix is a bit redundant. Should be either just `SOUND_EVENTS` or `SOUND_EVENTS_BY_<key type>`.